dev_guide/managing_images.adoc

```diff
@@ -0,0 +1,582 @@
+      - ENV=/opt/app-root/etc/scl_enable
+      - PROMPT_COMMAND=. /opt/app-root/etc/scl_enable
+      - RUBY_VERSION=2.2
+      ExposedPorts:
+        8080/tcp: {}
+      Image: d9c3abc5456a9461954ff0de8ae25e0e016aad35700594714d42b687564b1f51
+      Labels:
+        build-date: 2015-12-23
+        io.k8s.description: Platform for building and running Ruby 2.2 applications
+        io.k8s.display-name: Ruby 2.2
+        io.openshift.builder-base-version: 8d95148
+        io.openshift.builder-version: 8847438ba06307f86ac877465eadc835201241df
+        io.openshift.expose-services: 8080:http
+        io.openshift.s2i.scripts-url: image:///usr/libexec/s2i
+        io.openshift.tags: builder,ruby,ruby22
+        io.s2i.scripts-url: image:///usr/libexec/s2i
+        license: GPLv2
+        name: CentOS Base Image
+        vendor: CentOS
+      User: "1001"
+      WorkingDir: /opt/app-root/src
+    ContainerConfig: {}
+    Created: 2016-01-26T21:07:27Z
+    DockerVersion: 1.8.2-el7
+    Id: 57b08d979c86f4500dc8cad639c9518744c8dd39447c055a3517dc9c18d6fccd
+    Parent: d9c3abc5456a9461954ff0de8ae25e0e016aad35700594714d42b687564b1f51
+    Size: 430037130
+    apiVersion: "1.0"
+    kind: DockerImage
+  dockerImageMetadataVersion: "1.0"
+  dockerImageReference: centos/ruby-22-centos7@sha256:3a335d7d8a452970c5b4054ad7118ff134b3a6b50a2bb6d0c07c746e8986b28e
+  metadata:
+    creationTimestamp: 2016-01-29T13:17:45Z
+    name: sha256:3a335d7d8a452970c5b4054ad7118ff134b3a6b50a2bb6d0c07c746e8986b28e
+    resourceVersion: "352"
+    uid: af2e7a0c-c68a-11e5-8a99-525400f25e34
+kind: ImageStreamImage
+metadata:
+  creationTimestamp: null
+  name: ruby@3a335d7
+  namespace: openshift
+  selfLink: /oapi/v1/namespaces/openshift/imagestreamimages/ruby@3a335d7
+----
+====
+
+[[image-pull-policy]]
+== Image Pull Policy
+
+Each container in a pod has a Docker image. Once you have created an image and
+pushed it to a registry, you can then refer to it in the pod.
+
+When {product-title} creates containers, it uses the container's
+`*imagePullPolicy*` to determine if the image should be pulled prior to starting
+the container. There are three possible values for `*imagePullPolicy*`:
+
+- `*Always*` - always pull the image.
+- `*IfNotPresent*` - only pull the image if it does not already exist on the node.
+- `*Never*` - never pull the image.
+
+If a container's `*imagePullPolicy*`
+parameter is not specified, {product-title} sets it based on the image's tag:
+
+. If the tag is *latest*, {product-title} defaults `*imagePullPolicy*` to `*Always*`.
+. Otherwise, {product-title} defaults `*imagePullPolicy*` to `*IfNotPresent*`.
+
+[[importing-tag-and-image-metadata]]
+== Importing Tag and Image Metadata
+
+An image stream can be configured to import tag and image metadata from an image
+repository in an external Docker image registry. To create an image stream from an external registry, set the
+`spec.dockerImageRepository` field appropriately. For example:
```

Comment:
I'd mention also an alternative that lets you achieve the same without creating an image stream:

    oc import-image --from=openshift/ruby-20-centos7 --confirm ruby